practice-01/js/index.js

```diff
@@ -241,15 +263,26 @@ function displayFormResults() {
 formId.addEventListener('submit', function (e) {
     e.preventDefault()
 
-    validateEmpty();
-
-    validateEmail();
-
-    validateName();
-
-    validateCheckNumber();
-
-    validateTextLength();
-
+    if (validateEmpty() === false) {
+        validateEmpty();
+        return alert('Please enter full information Name , Email, Age ');
+    } else {
+        if(validateName() === false){
+            validateName();
+            return;
+        }
+        if(validateEmail() === false){
+            validateEmail();
+            return;
+        }

```

Comment:
I dont understand why we need to do that. 
``` js
if(validateEmail() === false){
            validateEmail();
            return;
        }

```

call validateEmail(); twice. Could you please find another solution